app/views/user_mailer/claim_notification.html.erb

```diff
@@ -1,17 +1,17 @@
 <% content_for :message do %>
   <h2 class="heading">
-    Hello from the Archive of Our Own!
+    <%= t '.hello' %>
   </h2>
 
-  <p>You're receiving this e-mail because you had works in a fanworks archive that has been imported by <%= style_link('Open Doors', 'http://opendoors.transformativeworks.org/') %> into the <%= style_link('Archive of Our Own', root_url) %> (AO3). Because this e-mail address is connected to one registered on the imported archive, the associated fanworks (listed below) have been automatically added to your AO3 account.</p>
+  <p><%= t( '.html.part1', open_doors: style_link(t('.open_doors'),'http://opendoors.transformativeworks.org'), ao3: style_link(t('.ao3'), root_url )).html_safe %></p>
 
-  <p>You can read announcements about recent archive moves at <%= style_link('AO3 News', 'http://archiveofourown.org/admin_posts?tag=18') %>, and find additional information on Open Doors' <%= style_link('FAQ page', 'http://opendoors.transformativeworks.org/faq') %> or <%= style_link('tutorials page', 'http://opendoors.transformativeworks.org/tutorials') %>. For any questions not answered in the FAQ, tutorials, or this e-mail, please <%= support_link('contact AO3 Support') %>.</p>
+  <p><%= t( '.html.part2', ao3_news: style_link(t('.ao3_news'), 'http://archiveofourown.org/admin_posts?tag=18'), open_doors_faq: style_link(t('.open_doors_faq'), 'http://opendoors.transformativeworks.org/faq'), tutorial_page: style_link(t('.tutorials_page'), 'http://opendoors.transformativeworks.org/tutorials'), support: support_link(t '.contact_support')).html_safe %></p>
```

Comment:
Two unnecessary spaces on this line:
* `t( '.html.part2',`
* `support_link(t '.contact_support')`